### PR TITLE
Service fixes

### PR DESCRIPTION
**Pull request by AllanT102**
🎟️ **Ticket(s):**
Closes #

👷 **Changes:**
- Went through all services to fix all dynamo db calls/
- tested most endpoints locally except for faved events lol

💭 **Notes:**
- Still needs additional testing in dev environment – should smoke test it

**Wait! Before you merge, have you checked the following:**
- [ ] Serverless tests are passing (Check travis build logs, CI is currently broken)
- [ ] PR is has approving review(s)

### CODE REVIEW

File: lib/db.js [hunk 1]
```diff
@@ -215,19 +213,24 @@ export default {
...
     try {
-      const res = await docClient.put(params).promise();
-      return res
+      const command = new PutCommand(params);
+      const res = await docClient.send(command);
+      return res;
     } catch (err) {
       const errorResponse = this.dynamoErrorResponse(err);
-      throw errorResponse
+      throw errorResponse;
```
Comment (voctory) [inline lib/db.js:233]: I'm curious about the searchability and formatting of this exception on the CloudWatch logs compared to error logging? I've found other projects will also have a console.error or similar before throwing an exception, so I'm not sure if we're benefiting from removing all our error logging - lmk

File: services/users/handler.js [hunk 2]
```diff
@@ -338,7 +331,9 @@ export const favouriteEvent = async (event, ctx, callback) => {
 
     let expressionAttributeValues;
     expressionAttributeValues = {
-      ":eventsIDAndYear": docClient.createSet([eventIDAndYear]) // set data type, for updateExpression
+      ":eventsIDAndYear": marshall({
```
Comment (voctory) [inline services/users/handler.js:334]: I'm not familiar with marshalling, but just wanted to check if we need it here? Our docClient uses lib-dynamodb, which is supposed to automatically perform it https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-util-dynamodb/

**Review by voctory — APPROVED** [review]
Left 2 comments asking for clarification, will approve tho

File: services/users/handler.js (hunk 2 above)
Comment (AllanT102) [reply services/users/handler.js:334]: Yeah looked into it. I think either works, but createSet is preferred for dynamic languages. 

Screenshot 2024-08-12 at 12 20 57 AM

File: lib/db.js (hunk 1 above)
Comment (AllanT102) [reply lib/db.js:234]: oops didnt mean to delete the ddb error log

**Pull request merged**